main/.gcloudignore

```diff
@@ -0,0 +1,20 @@
+#*#
+*/RCS/*
+*.bak
+*.pyc
+*.pyo
+*~
+Icon\r
+.*
+app.yaml
+app.yml
+index.yaml
+index.yml
+lib/*
+static/dev/*
+static/ext/.*.coffee
+static/ext/.*.css
+static/ext/.*.js
+static/ext/.*.less
+static/ext/.*.json
+static/src/.*
```

Comment:
is that supported in the current version? Maybe we could make this change in the master?

Reply:
Perhaps, but a quick google for gae .gcloudignore didn't answer the question. It's a change you have to make for python 3 environment though.